### PR TITLE
Remove list-shares.

### PR DESCRIPTION
**Pull request by howbazaar**
Addresses http://pad.lv/1623097. User info is shown with show-model, list-shares is to be removed.

(Review request: http://reviews.vapour.ws/r/5670/)

### CODE REVIEW

**Comment by howbazaar** [comment]
$$merge$$

File: cmd/juju/commands/main.go [hunk 1]
```diff
@@ -305,7 +305,6 @@ func registerCommands(r commandRegistry, ctx *cmd.Context) {
 	r.Register(model.NewDefaultsCommand())
 	r.Register(model.NewRetryProvisioningCommand())
 	r.Register(model.NewDestroyCommand())
-	r.Register(model.NewUsersCommand())
```
Comment (natefinch) [inline cmd/juju/commands/main.go]: Good bye!

File: featuretests/cmd_juju_model_test.go [hunk 2]
```diff
@@ -81,30 +81,6 @@ func (s *cmdModelSuite) TestRevokeModelCmdStack(c *gc.C) {
 	c.Assert(modelUser, gc.DeepEquals, description.UserAccess{})
 }
 
-func (s *cmdModelSuite) TestModelUsersCmd(c *gc.C) {
```
Comment (natefinch) [inline featuretests/cmd_juju_model_test.go]: yay deleting code!

**Comment by howbazaar** [comment]
$$merge$$

**Pull request merged**